core/templates/dev/head/pages/exploration_editor/simple_editor_tab/questions/MultipleChoiceEditorDirective.js

```diff
@@ -41,6 +41,15 @@ oppia.directive('multipleChoiceEditor', [function() {
         // Note that a questionId generated in this way may contain spaces,
         // since it is just the state name.
         $scope.questionId = $scope.getUniqueId();
+
+        $scope.getHash = function(suffix) {
```

Comment:
The purpose is just to guarantee a unique id,  it creates a unique id for sub-elements of a question based on the __questionId__. I mainly added it to avoid long javascript in the html.